wiki/FAQ/en.md

```diff
@@ -274,71 +358,126 @@ _Detailed explanation of scoring can be found at the game style's own respective
-:---: | :---
-![osu! smoke effect][osu_smoke image] | Usage of "Smoke"
-![osu! smoke settings][osu_smoke_set image] | "Smoke" in key bindings settings
+_Top image shows a slider with all the sliderticks shown in Edit mode; Bottom image shows the same slider during play, with some sliderticks obstructed by start and end circle of the slider._
 
-### Why do I not have the max possible combo when I have no misses?
-You released a slider too early or you barely clicked the hit-circle. [Answer's Source][reddit osugame wiki]
+**You released a slider early (meaning that you released the ball before it had finished the ending slider point).**
+
+A slider is composed of a starting circle point, spaced smaller circles known as sliderticks, repeat points (if used, it will be treated as starting point instead; it is a proper slider in reverse), and an ending circle point. Each of those points on a slider will add one to your combo. Note that the starting circle point is worth 30 points, the sliderticks are worth 10 points each, the repeat points are worth 30 points each, and the end circle point is worth 300 (or geki), 100 (or katu), or 50 points depending on how the slider was completed.
+
+Sliderticks only appears inside its own sliderpath; if the sliderticks are too close to the start or end circle as shown in the image above, it will be hidden inside the start or end circle. Hidden sliderticks still counts into combo and score, so it must be collected along with the visible sliderticks. If the slider has one or more repeat points, then the sliderticks will reappear at the same place to be collected in reverse.
+
+If miss the timing to tap the starting circle point, miss the sliderticks, or miss the repeat points, combo will break (it should trigger a combo break sound when the combo is high enough) but it is not considered a miss. However, if miss the ending circle point, combo does not break and does not increment by 1. It is also not considered a miss; instead, a 100 (or katu for end of the combo) will be given for the entire slider, assuming at least started the slider properly, and completed all the sliderticks/repeat points when given. Notably, [Sudden Death][Sudden Death wikilink] game mod will not trigger its effect of immediate game over when missing the ending circle point (since the combo did not break, the Sudden Death effect was not triggered).
+
+These situations, caused by a slider, are collectively called sliderbreak. A max combo play can be known by checking the result screen for the _Perfect_ notice on the performance graph; not having the _Perfect_ notice, especially without misses, usually means that one or more sliderbreak occured during play.
+
+[Answer source with test example in gaming section of stackexchange](https://gaming.stackexchange.com/questions/221663/i-fcd-a-song-why-is-my-combo-lower-than-people-on-the-high-scores)
+
+### osu!taiko
+#### What does Use Taiko skin for Taiko mode in Options under Skin section does?
```

Comment:
change header to 
``What does Use Taiko skin for Taiko mode in Options under Skin section do?``

I think you should also quote the button's name? (because the heading reads kinda oddly without them??)